dandi/dandiapi.py

```diff
@@ -425,6 +425,10 @@ def publish(self) -> "RemoteDandiset":
         lgr.debug("Waiting for Dandiset %s to complete validation ...", self.identifier)
         while True:
             r = self.client.get(f"{self.version_api_path}info/")
+            if "status" not in r:
```

Comment:
I guess it is ok in short term, but I hope that @waxlamp @dchiquito  would consider addressing https://github.com/dandi/dandi-api/issues/368 so there is no guess work in interactions with the API server.